GovUk.Frontend.ExampleApp/wwwroot/css/site.css

```diff
@@ -1,3 +1,4 @@
 ﻿header { border-bottom: 1px solid #ccc; margin-top: 1rem; }
```

Comment:
If we're overriding the header, this line can go - it's pushing everything down anyway